scipy/optimize/_qap.py

```diff
@@ -13,26 +13,24 @@ def quadratic_assignment(
     Solve the quadratic assignment problem.
 
     This function solves the Quadratic Assignment Problem (QAP) and the
-    Graph Matching Problem through an implementation of the Fast
-    Approximate QAP Algorithm (FAQ) (these two problems are the same up
-    to a sign change) [1]_.
+    Graph Matching Problem (GMP) using the Fast Approximate QAP Algorithm
+    (FAQ) [1]_.
 
-    Quadratic Assignment solves problems of the following form:
+    Quadratic assignment solves problems of the following form:
 
     .. math::
 
-        \min_P & \ {\ \text{trace}(APB^T P^T)}\\
+        \min_P & \ {\ \text{trace}(A^T P B P^T)}\\
         \mbox{s.t. } & {P \ \epsilon \ \mathcal{P}}\\
 
     where :math:`\mathcal{P}` is the set of all permutation matrices,
     and :math:`A` and :math:`B` are adjacency matrices.
 
-    This algorithm can be thought of as finding an alignment of the
-    vertices of two graphs which minimizes the number of induced edge
+    Graph matching tries to *maximize* the same objective function.
+    This algorithm can be thought of as finding the alignment of the
+    nodes of two graphs that minimizes the number of induced edge
```

Comment:
"vertices" was used in 2 places, "nodes" was used in 5. Let's stick with "nodes" for consistency.